README.md

```diff
@@ -87,41 +71,34 @@ Further, prerequisites might be necessary for specific tasks. These are listed u
 
 ### Installation
 
-You have two options to use this project:
- - as a pure Python project
- - using the _start scripts_ which pull the correct container image from Dockerhub and execute it within the Docker container
+#### PyPi - recommend
+
+```
+python3 -m pip install exasol-script-languages-container-tool
+```
 
-#### Pure Python
+#### From Github release
```

Comment:
Not needed anymore, I would say

Reply:
I was in doubt if we should keep it. I remove it then...